config/samples/dataplane_v1beta1_openstackdataplanenodeset_networker.yaml

```diff
@@ -20,15 +20,20 @@ spec:
         hostName: edpm-networker-0
         ansible:
           ansibleHost: 192.168.122.100
-          ansibleVars:
-            ctlplane_ip: 192.168.122.100
-            internalapi_ip: 172.17.0.100
-            storage_ip: 172.18.0.100
-            tenant_ip: 172.19.0.100
-            fqdn_internalapi: edpm-networker-0.example.com
   networkAttachments:
     - ctlplane
   nodeTemplate:
+    networks:
```

Comment:
same here move to 'nodes` section as you can't have `fixedIP` in a template for all nodes.